lms/services/assignment.py

```diff
@@ -32,6 +32,27 @@ def get_assignment(self, tool_consumer_instance_guid, resource_link_id):
             .one_or_none()
         )
 
+    def get_copied_from_assignment(self, lti_params) -> Optional[Assignment]:
```

Comment:
Not sure about this name

Reply:
My first thought was `get_original_assignment()` but "original" might imply the very first original which I'm not sure this necessarily is.

Alternative: `get_historical_assignment()` to match the the wording used in the `resource_link_id_history`, `ext_d2l_resource_link_id_history` and `custom_ResourceLink.id.history` param names.

Reply:
I like the `copied` part of the name, not 100% convinced though.